core/templates/dev/head/pages/exploration-editor-page/improvements-tab/improvements-tab.directive.ts

```diff
@@ -70,6 +70,14 @@ angular.module('oppia').directive('improvementsTab', [
             return ImprovementsDisplayService.isOpen(card.getStatus());
           };
 
+          $scope.getCardCssClasses = function(card) {
```

Comment:
In general, try not to construct CSS or HTML strings in JS. I think you can pass in a dict to ng-class -- the keys are the CSS class names and the values are conditions which evaluate to booleans.